### PR TITLE
Breaking

### PR DESCRIPTION
**Pull request by notlee**
See https://github.com/Financial-Times/o-topper/issues/36

### CODE REVIEW

**Comment by origamiserviceuser** [comment]
o-topper bundle size difference from 2.7.2 to 33ad1ec8deb3a666c283ee8ea3d02268c87d1158
js: 0.33kb increase (0.08kb/gzip)
css, master: 0.63kb decrease (-0.01kb/gzip)

**Comment by origamiserviceuser** [comment]
o-topper bundle size difference from 2.7.2 to 1881a62e7920a1a87c502a61a5124e43b0e560f4
js: 0.33kb increase (0.08kb/gzip)
css, master: 0.63kb decrease (-0.01kb/gzip)

**Comment by notlee** [comment]
> The last piece of work that needs to happen is to update `o-editorial-typography` once that's released. I don't feel confident that I know what needs to happen in that repo before we release 😬

[Nothing special outstanding](https://github.com/Financial-Times/o-editorial-typography/pull/12)!

File: bower.json [hunk 1]
```diff
@@ -5,10 +5,11 @@
     "main.js"
   ],
   "dependencies": {
-    "o-typography": "^5.4.2",
-    "o-grid": "^4.3.7",
-    "o-colors": "^4.1.5",
-    "o-icons": "^5.6.0",
+    "o-typography": "^6.0.0",
+    "o-editorial-typography": "1.0.0-beta.2",
```
Comment (rowanmanning) [inline bower.json]: Thought a suggestion might be a useful way to get this done. Can be committed once editorial-typography is released.
```suggestion
    "o-editorial-typography": "^1.0.0",
```

**Comment by origamiserviceuser** [comment]
o-topper bundle size difference from 2.7.2 to 112c767f94948b143266e6eead8c9af1fd4f2a61
js: 0.28kb increase (0.07kb/gzip)
css, master: 0.62kb decrease (0.00kb/gzip)

**Pull request merged**